Sources/BarcodeScannerController.swift

```diff
@@ -300,15 +309,14 @@ open class BarcodeScannerController: UIViewController {
     flashButton.alpha = alpha
     settingsButton.isHidden = status.state != .unauthorized
   }
-
+  
   // MARK: - Layout
-  open override func viewWillLayoutSubviews() {
```

Comment:
I made this change because when I checked out on master I noticed that the focus view was not appearing the right size and on the tag 2.0 the focus view was working all right. Then, I compared both codes and noticed that the code was moved from `viewWillAppear` to `viewWillLayoutSubviews`. I thought that would maybe the reason, considering that when `viewWillLayoutSubviews` is called, we don't have a ready view yet. When I made the change, everything worked all right as it was working before.
<img width="1680" alt="screen shot 2017-07-13 at 13 20 34" src="https://user-images.githubusercontent.com/1952654/28176633-92801694-67ce-11e7-9cc9-82e14b49d789.png">